components/TextFields/tests/snapshot/supplemental/MDCAbstractTextFieldSnapshotTests+I18N.m

```diff
@@ -28,8 +28,15 @@ - (void)MDCForceTextInputRightToLeft:(UIView *)view {
     if (textRange) {
       [textInput setBaseWritingDirection:UITextWritingDirectionRightToLeft forRange:textRange];
     } else {
-      // Fail the test if `textRange` is nil since even an empty range should be non-nil.
-      XCTAssertNotNil(textRange);
+      if (@available(iOS 11.0, *)) {
+        // Fail the test if `textRange` is nil since even an empty range should be non-nil.
+        XCTAssertNotNil(textRange);
+      } else {
+        // iOS before 11 would return `nil` for `beginningOfDocument` unless the UITextInput was
```

Comment:
do you think there is an alternative for making this work for pre iOS 11? Also, what does this mean for Snapshots of previous iOS versions, will they just not be recorded?

Reply:
No, the text will just be aligned to the simulator's locale alignment. Our current tests would only be able to run on iOS 10+ in any case (due to the use of `UIGraphicsImageRenderer`). I suspect that if we wanted to run snapshot tests on iOS 10 for some reason, we would have to either find a work-around for this or just skip these particular tests on that iOS version.